src/servers/main.cc

```diff
@@ -553,6 +610,14 @@ Parse(nvidia::inferenceserver::InferenceServer* server, int argc, char** argv)
       case OPTION_TF_GPU_MEMORY_FRACTION:
         tf_gpu_memory_fraction = ParseFloatOption(optarg);
         break;
+      case OPTION_TF_ADD_VGPU:
+        bool malformed;
+        std::pair<int, std::vector<float>> map_element =
+            ParseVGPUOption(optarg, &malformed);
+        if (!malformed) {
+          tf_vgpu[map_element.first] = map_element.second;
```

Comment:
You can't just continue... if something went wrong with the parsing then you should return false.